sfdx/oclif/mobilesdk/android/create.js

```diff
@@ -0,0 +1,48 @@
+/*
```

Comment:
With Oclif commands are no longer defined progmatically instead it's now declarative base on the filesystem layout. Directories refer to topics and files to command. The wiring now happens in package.json::oclif. There could be some refactoring to optimize the code more but I'll leave that to you guys.